docs/docs/development/hardware-integration/encoders.md

```diff
@@ -0,0 +1,80 @@
+---
+title: EC11 Encoders
+sidebar_label: Encoders
+---
+
+EC11 encoder support can be added to your board or shield by adding the appropriate lines to your board/shield's configuration (.conf), device tree (.dtsi), overlay (.overlay), and keymap (.keymap) files.
+
+## Configuration File
+
+In your configuration file you will need to add the following lines so that the encoders can be enabled/disabled:
+
+```ini
+# Uncomment to enable encoder
+# CONFIG_EC11=y
+# CONFIG_EC11_TRIGGER_GLOBAL_THREAD=y
+```
+
+These should be commented by default for encoders that are optional/can be swapped with switches, but can be uncommented if encoders are part of the default design.
+
+:::note
+If building locally for split boards, you may need to add these lines to the specific half's configuration file as well as the combined configuration file.
+:::
+
+## Devicetree File
+
+In your device tree file you will need to add the following lines to define the encoder sensor:
```

Comment:
```suggestion
In your devicetree file you will need to define each sensor with their properties. For split keyboards, do this in the .dtsi file that is shared by all parts; otherwise do it in the .dts (for boards) or .overlay (shields) file, see [configuration overview](../../config/index.md#devicetree-files) for details. Add the following lines:
```